nusight2/src/client/components/localisation/robot_model.ts

```diff
@@ -158,7 +162,8 @@ export class LocalisationRobotModel {
       Hfw: Matrix4.of(),
       Rwt: Quaternion.of(),
       motors: ServoMotorSet.of(),
-      fieldLinesDots: { rPWw: [] },
+      fieldLinePoints: { rPWw: [] },
+      ball: { rBWw: Vector3.of() },
```

Comment:
```suggestion
```

Reply:
In `network.ts` where we update `onBall` I am getting a warning `robot.ball is possibly undefined`, do I need to do somewhere there as well now that ball is optional?

Reply:
Update the whole thing:

```ts
robot.ball = { rBWw: Vector3.from(ball.rBWw) };
```